apps/dashboard/src/main.tsx

```diff
@@ -3,18 +3,61 @@ import * as ReactDOM from "react-dom/client"
 import { BrowserRouter } from "react-router-dom"
 import { ChakraProvider } from "@chakra-ui/react"
 import { CookiesProvider } from "react-cookie"
-
+import {
+    connectorsForWallets,
+    RainbowKitProvider
+} from "@rainbow-me/rainbowkit"
+import { configureChains, createClient, WagmiConfig } from "wagmi"
+import { goerli } from "wagmi/chains"
+import { publicProvider } from "wagmi/providers/public"
+import {
+    metaMaskWallet,
+    coinbaseWallet,
+    walletConnectWallet,
+    trustWallet,
+    ledgerWallet,
+    injectedWallet
+} from "@rainbow-me/rainbowkit/wallets"
 import App from "./app/app"
+import theme from "./styles"
+
+const { chains, provider, webSocketProvider } = configureChains(
+    [goerli],
+    [publicProvider()]
+)
+
+const connectors = connectorsForWallets([
+    {
+        groupName: "Wallets",
+        wallets: [
+            injectedWallet({ chains }),
+            metaMaskWallet({ chains }),
+            coinbaseWallet({ appName: "Zk Groups", chains }),
+            walletConnectWallet({ chains }),
```

Comment:
We can add more options, but I realized some wallets don't support goerli - list Trust Wallet. So need to test each before adding.
Anyway WalletConnect is supported by most wallets, so should be good for now.